developer/installation/installation-osx.md

```diff
@@ -0,0 +1,139 @@
+# Installation for OSX
+
+[![Installation Video](/assets/guide-installation-video-screenshot.png)](https://www.youtube.com/watch?v=PkFDX8NWskY)
+
+## Install prerequisites:
+
+### Install node
+Download and run the installer from the [NodeJs site](https://nodejs.org)
+
+### Install Xcode
+Download and run the installer from [Apple Developer Site](https://developer.apple.com/download/)
+
+Then from the command line run
+
+```sh
+xcode-select --install
+```
+
+### Install Homebrew
+
+(If you are uncomfortable just running a random script from Github, you can also visit the [Homebrew site](http://brew.sh/))
```

Comment:
Just drop line 21 entirely and tell people to go to https://brew.sh/

Reply:
I did this on purpose because I want this to all be in one place. Most people are just going to cut and paste.